### DISCUSSION TITLE
Help with RX 6650 XT on ROCm & How Can I Contribute?

### DISCUSSION THREAD

**Discussion by andersprenger** [ROCm/ROCm#2717] [Q&A] [answered]
Hello ROCm Community, 👋

I'm working with an RX 6650 XT and aiming to use it for AI development, but I'm encountering challenges in running tasks without experiencing segmentation faults. I'm seeking advice on how to effectively use this GPU with ROCm for AI, particularly with TensorFlow and PyTorch.

1. **RX 6650 XT for AI Development**: 
   - Has anyone successfully used the RX 6650 XT for AI development with ROCm? I would greatly appreciate any insights on settings, drivers, or specific ROCm versions that work well with this GPU. 
   - If you've encountered and resolved segmentation faults with this setup, how did you do it?

2. **Best Practices and Configuration**:
   - What are the recommended practices or configurations for using the RX 6650 XT in AI development environments? Are there particular tweaks or settings in ROCm that enhance stability and performance?

3. **Community Experiences and Suggestions**:
   - I would love to hear from anyone who has navigated similar issues or has tips on optimizing the RX 6650 XT for TensorFlow and PyTorch under ROCm, on either Windows or Linux.

4. **Contributing to ROCm**: 
   - What areas of the ROCm project could benefit from more community involvement? As a computer science student, what skills are most relevant for contributing?

Looking forward to your insights. Thanks for your help.

Best,
Anderson Sprenger

**Comment by renwenye** [comment]
segment fault as well!!
As a user of 6650xt, i really also need these info! i've been wasting a lot of my time for this , and failed.

**Comment by darren-amd** [comment, marked as answer]
Hi @andersprenger @renwenye ,

Glad to hear that you're eager to contribute. Although we do not currently support the RX 6650 XT (see [Support Matrix](https://rocm.docs.amd.com/projects/install-on-linux/en/latest/reference/system-requirements.html)), I did find a few users who were able to get it working. Could you try setting the following environment variables and give it another try?
```
export AMDGPU_TARGETS="gfx1032"
export HSA_OVERRIDE_GFX_VERSION=10.3.0
```

Regarding contributing, I think we would most benefit from your feedback on where we can improve our current stack of tools, as well as highlighting any bugs/issues that you may encounter. You can find more information on contributing here: [Contribute to ROCm](https://github.com/ROCm/ROCm/blob/develop/CONTRIBUTING.md)
